pkg/gotohelm/transpiler.go

```diff
@@ -492,6 +503,65 @@ func (t *Transpiler) transpileStatement(stmt ast.Stmt) Node {
 	})
 }
 
+// transpileMVAssignStmt handles transpiling assignments where the RHS has
+// exactly one expression and the LHS has more than one. E.g type checks, map
+// checks, and functions with multiple return values.
+func (t *Transpiler) transpileMVAssignStmt(stmt *ast.AssignStmt) Node {
+	var stmts []Node
+
+	// We need an intermediate variable to assign to. We'll synthesize it from
+	// the identifiers on the LHS and position of the AssignStmt to get something that is unique,
+	// deterministic, and mildly human readable.
+	// foo, ok := ... -> $_123_foo_ok := ...
+	intermediate := &Ident{Name: fmt.Sprintf("_%d", stmt.Pos())}
```

Comment:
I'm not sure why, but the `stmt.Pos()` is not stable. When this would be added to CI it will fail.

Reply:
Do you have an example failure I could see?

Reply:
Just re-run
```
go test ./pkg/gotohelm -v 
```